src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2824,34 +2837,250 @@ get_var_dfd (OstreeSysroot      *self,
   return glnx_opendirat (base_dfd, base_path, TRUE, ret_fd, error);
 }
 
+/* Get a GFile* referring to the commit object at ref for the file at path. */
```

Comment:
For my understanding, does this mean "Get a GFile* that points to the file at path in the content for ref"?